examples/account/create_vp.rs

```diff
@@ -81,26 +125,29 @@ async fn main() -> Result<()> {
   // Create an unsigned Presentation from the previously issued Verifiable Credential.
   let mut presentation: Presentation = PresentationBuilder::default()
     .id(Url::parse("asdf:foo:a87w3guasbdfuasbdfs")?)
```

Comment:
@olivereanderson and I think we can remove this, because we don't have a sensible value to set. The `id` field is optional and since we don't need to refer to the presentation again, we don't need a unique identifier here.

While we're on the topic, we also agreed that it would be good to update the documentation of `PresentationBuilder::id` to `/// Sets the unique identifier of the presentation`, to be more expressive and in-line with the spec. Would you mind including that in this PR?